_data/utilities.yml

```
@@ -436,8 +436,9 @@ websites:
     tfa:
       - sms
       - totp
-      - hardware
+      - proprietary
```

The proprietary T-Mobile app is not the authentication factor here, but rather biometric features. These are currently classified as _hardware_.
```suggestion
      - hardware
```